dawn/renderer/actions/AlertActions.js

```diff
@@ -12,7 +12,7 @@ export const addAsyncAlert = (heading, message) => ({
   message,
 });
 
-export const removeAsyncAlert = (id) => ({
+export const removeAsyncAlert = id => ({
```

Comment:
Linter again; [rule](http://eslint.org/docs/rules/arrow-parens) specifying no parentheses around single argument if what's coming after it isn't braces.